tests/system/small/test_dataframe_io.py

```diff
@@ -150,7 +153,7 @@ def test_to_csv_index(
     # read_csv will decode into bytes inproperly, convert_pandas_dtypes will encode properly from string
     dtype.pop("bytes_col")
     gcs_df = pd.read_csv(
-        path,
+        path.replace("*", FIRST_GCS_FILE_SUFFIX),
```

Comment:
[nit] would be neat to abstract it out in a function in utils:

```python
def get_first_file_from_wildcard(path):
    return path.replace("*", FIRST_GCS_FILE_SUFFIX)
```

Reply:
Done.